core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -706,4 +707,17 @@ private SqlLibraryOperators() {
   @LibraryOperator(libraries = { POSTGRESQL })
   public static final SqlOperator INFIX_CAST =
       new SqlCastOperator();
+
+  /** Null-safe "&lt;=&gt;" equal operator used by MySQL, for example
+   * {@code 1<=>NULL}. */
+  @LibraryOperator(libraries = { MYSQL })
+  public static final SqlOperator NULL_SAFE_EQUAL =
```

Comment:
I think The NULL_SAFE_EQUAL **is** IS_NOT_DISTINCT_FROM. Not belong to. So

```
public static final SqlOperator NULL_SAFE_EQUAL = SqlStdOperatorTable.IS_NOT_DISTINCT_FROM;
```
right ?

Reply:
Yes, you are right.
I update it.
Thanks.

Reply:
Why do we need to change the operator's name? If we use the SqlStdOperatorTable.IS_NOT_DISTINCT_FROM directly, Then can support parse and execute. Now only can support parse. If I miss something please let me know. 
and Add another parse test cases for `(a, b) <=> (x, y)`

Reply:
@NobiGo 
When using SqlStdOperatorTable#IS_NOT_DISTINCT_FROM, '`X` <=> 3' will return '`X` IS NOT DISTINCT FROM 3'.
I think it destroys the user's original input.

Reply:
According to [MySQL '<=>'](https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#operator_equal-to) . We should cover it.

Reply:
Please don't force fush commit, until we have a final state.

Reply:
@NobiGo 
Ok, got it.
In my project, I didn't meet these complex scenes.
I will add some related test cases.
Thanks a lot.